protocol/app/app.go

```diff
@@ -429,6 +443,14 @@ func New(
 		sdk.GetConfig().GetBech32AccountAddrPrefix(),
 		lib.GovModuleAddress.String(),
 	)
+
+	app.AuthzKeeper = authzkeeper.NewKeeper(
+		runtime.NewKVStoreService(keys[authzkeeper.StoreKey]),
+		appCodec,
```

Comment:
the order of these params is also reversed vs most `NewKeeper` methods?

Reply:
yeah `authz` always have store key before codec, not sure how it was bootstrapped